### DISCUSSION TITLE
Zwift zwo file - is it possible to include incline in a zwo file for workout?

### DISCUSSION THREAD

**Discussion by ogperalta** [cagnulein/qdomyos-zwift#1342] [General]
Hello, I have created a workout on www.zwiftworkout.com and saved it in a folder and use it from the "open training program" option. it works perfectly. however, I would like to create some workouts that include incline. from www.zwiftworkout.com it is not possible to add this option and I was wondering if it was possible to add it directly in the zwo file as one more parameter
For example
<SteadyState Duration="90" Power="0.5" pace="1" Inclination="3"/>
Is this possible? how can I do it?

**Comment by cagnulein** [comment]
@ogperalta yes check this https://github.com/cagnulein/qdomyos-zwift/issues/1209#issuecomment-1421886702 @victorypoint

let me know

**Comment by ogperalta** [comment]
Hello @cagnulein , I have a RunnerT. with 1, 2 and 3% it works, but with 4% onwards there is no change in inclination...in fact as a curious thing, when the workout starts it always shows 1% even though it is at 0 on the treadmill. In RunnerT Options i have ckecked Fitfiu MC-460

**Reply by cagnulein** [reply]
send a debug log https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F and i will check it
